content/en/logs/workspaces/use_cases/correlate_access_error_logs.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Correlate Web Access Logs and Web Error Logs
+further_reading:
+- link: "/logs/workspaces/"
+  tag: "Documentation"
+  text: "Learn more about Log Workspaces"
+---
+
+## Use case
+
+Log Workspaces enable you to integrate and correlate Apache Web Access Logs and Web Error Logs, providing clear insights into how specific errors impact users. Analyse these logs togehter to identify which user actions cause errors and address issues proactively. Follow these steps to monitor and correlate Apache Web access and Error logs using the flexible querying and visualization options available within Workspaces.
+
+## Setup
+
+This guide assumes that you are:
+- Submitting logs to Datadog for a similar use case.
+- Able to [create a workspace][1] and add cells. 
+
+### 1. Bring in your data source
+
+To get started, bring in the logs from the service(s) you want to analyze.
+1. [Create a new Workspace][1].
+1. Select **Logs Query** as your data source.
+
+### 2. Query for failed logins
+
+Set up your logs query to filter for `http_web_access` logs and `web_error_logs` to search both sets of logs and gain a better understanding of user errors. For example,
+
+{{< img src="/logs/workspace/use_cases/correlate_access_error_logs/query_access_error_logs.png" alt="Query to filter for http_web_access and web_error_logs" style="width:100%;" >}}
+
+You can add any additional filters, facets, or attributes to narrow your search based on your requirements and what is available in your logs.
+
+### 3. Group status codes together
+
+To analyze the data further, use the Analysis feature to query your datasets with SQL. Create a `status_group` by 200s, 300s, 400s, and 500s to group the status codes together. Anything not matching a status code is marked "unknown". This is helpful for correlating the status codes to the user errors as we will see later on. 
+
+1. Add an [Analysis cell][3] to your workspace.
+1. Run a SQL query.
```

Comment:
I'm wondering if "a" is the best word to use here, or if "this" query (followed by a colon) might be more clear?

Reply:
```suggestion
1. Run this SQL query.
```